.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "mbed-os"]
     path = mbed-os
-    url = https://github.com/ARMmbed/mbed-os.git
+    url = https://github.com/uwrobotics/mbed-os.git
```

Comment:
This relies on a feature that still has some issues. This is a fork of the repo that automatically updates the master and feature-cmake-object-libraries branches(check out the auto-sync branch's `.github/pull.yml`). It _should_ also automatically **rebase** the `feature-cmake-object-libraries-with-stm32-fix` branch which i created (and this pr's submodule is pinned to the head of) which is just the latest feature-cmake-object-libraries with a  commit from Arm mbed-os's master that fixes all the stm32 builds they broke with the mbed 6.9 release